### DISCUSSION TITLE
Select the language to transcribe

### DISCUSSION THREAD

**Discussion by bcastaing** [openai/whisper#1852] [General]
Hello all, my audio contains english and french language. Is there a way to select the language to transcribe ? I want french only.
I did a search, and nothing exactly on this subject, sorry if I missed.
Thanx

**Comment by glangford** [comment]
From the command line `--language fr`

**Comment by bcastaing** [comment]
I tried this option but still translate english in french and I want to exclude the english talk.
The audio is an english speech with the translator so the translation of english talk give confusion.

**Reply by glangford** [reply]
It sounds like you are running into the "accidental" feature of `whisper` occasionally translating into languages other than English. I don't know of a workaround, but you can see some past discussion here

* https://github.com/openai/whisper/discussions/49
